concordium-sdk/src/main/java/com/concordium/sdk/ClientV2MapperExtensions.java

```diff
@@ -0,0 +1,79 @@
+package com.concordium.sdk;
+
+import com.concordium.grpc.v2.ArInfo;
+import com.concordium.grpc.v2.Empty;
+import com.concordium.sdk.crypto.elgamal.ElgamalPublicKey;
+import com.concordium.sdk.requests.BlockHashInput;
+import com.concordium.sdk.responses.blocksummary.updates.queues.AnonymityRevokerInfo;
+import com.concordium.sdk.responses.blocksummary.updates.queues.Description;
+import com.concordium.sdk.transactions.Hash;
+import com.google.common.base.Function;
+import com.google.common.collect.Iterators;
+import com.google.protobuf.ByteString;
+import lombok.var;
+
+import java.util.Iterator;
+import java.util.Objects;
+
+/**
+ * Object Mapping Extensions. Maps from GRPC types to client types and vice versa.
+ */
+interface ClientV2MapperExtensions {
+    static com.concordium.grpc.v2.BlockHashInput to(final BlockHashInput input) {
```

Comment:
Where is this used?

Reply:
https://github.com/Concordium/concordium-java-sdk/blob/8d6547250d9c845f98511b464c34ec29c5564e76/concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java#L60